recipe/meta.yaml

```diff
@@ -1,33 +1,51 @@
-{% set version = "301" %}
+{% set version = "302" %}
 
 package:
   name: pywin32
   version: {{ version }}
 
 source:
   url: https://github.com/mhammond/pywin32/archive/b{{ version }}.tar.gz
-  sha256: 5366613c366c66b521a2e79780dcdf347310bd2b75022e424df320760931999c
+  sha256: 47fd53c0561ff3019043e86880597aacbd736910d8add485117d8ad055441927
   patches:
     - disable_exchange_modules.patch
     - 0003-remove-PATH-pth-hack.patch
 
 build:
-  number: 0
+  number: 2
   script:
-    - {{ PYTHON }} setup.py -q install --record=record.txt --skip-verstamp
-    - copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %PREFIX%\Lib\site-packages\win32\
-    - copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %LIBRARY_BIN%\
-  skip: true  # [not win or py27]
+    - |                                                                                           # [win and python_impl != 'pypy']
```

Comment:
(Is it "not needed for PyPy" or "only needed for CPython"? If the latter, then I'd write `[win and python_impl == 'cpython']` to make things clear if we'd ever package for another Python implementation.)

Reply:
well basically pywin32 doesn't work on pypy